README.md

```diff
@@ -257,6 +257,29 @@ const record = await linnia.addRecord(dataHash, metadata, dataUri, ethParams);
 
 
 
+# Attest data on Linnia
+
+Attest data on Linnia means sign a record and verify that contain legitimate and correct information.
+
+## linnia.signRecord
+
+```javascript
+linnia.signRecord(dataHash, ethParams);
+```
+
+Add record to Linnia
+
+### Parameters
+
+1. `String` - The data hash, hex-encoded, 0x prefixed
+2. `Object` - The ethParams ethereum account params. 
+
+### Returns
+
+`Promise<Record>` - A promise when resolved returns the record object that was signed.
```

Comment:
Shouldn't this return the attestation that was created?

Reply:
There is no such thing as an attestation. In the getAttestation method, we are currently returning a boolean. I could return a boolean but I thought to return the record with the new sigCount and the new IRIS score would be better

Reply:
I think the boolean makes more sense here. Thoughts? @anujshah108 @godfreyhobbs